src/platforms/common/enriching-events/scopes.mdx

```diff
@@ -101,3 +101,44 @@ attach any context information at all by calling `clear` on the scope, while the
 
 Keep in mind that `with-scope` will not capture any exceptions that happen inside its callback function,
 and every error that happens there will be silently ignored and *not* reported.
+
+<PlatformSection supported={["java", "android"]}>
```

Comment:
```suggestion
<PlatformSection supported={["java"]}>
```

Android is single hub already, actually maybe android inherits from `java` already, maybe we need to add non support for it

Reply:
👍